### PR TITLE
Modifications in pxc-package-testing job for ps80 Jenkins Server

### PR DESCRIPTION
**Pull request by panchal-yash**
Add logs backup playbook for pxc-package-testing

Modify the AMIs for PS80 Jenkins server support for pxc-package-testing

### CODE REVIEW

File: molecule/pxc/playbooks/logsbackup.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - set_fact:
+        jenkinsworkspace={{ lookup('ansible.builtin.env', 'JENWORKSPACE') }}
+      delegate_to: localhost
+
+    - name: Zip the logs
+      community.general.archive:
+        path: /var/log
+        dest: /var/log/{{man_ip}}_logs.tar.gz
+
+    - name: List the zipped logs
```
Comment (eleo007) [inline molecule/pxc/playbooks/logsbackup.yml:22]: I think we can skip listing the logs not to overload the logs with output

File: molecule/pxc/playbooks/logsbackup.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    - debug: var=listlogs
+
+    - name: Fetch the log zip to the main node
+      become: true
+      ansible.builtin.fetch:
+        src: /var/log/{{man_ip}}_logs.tar.gz
+        dest: "{{jenkinsworkspace}}/PXC/{{man_ip}}/"
+
+    - name: List the fetched logs
```
Comment (eleo007) [inline molecule/pxc/playbooks/logsbackup.yml:34]: The same

File: molecule/pxc/playbooks/prepare.yml [hunk 3]
```diff
@@ -5,6 +5,7 @@
   become_method: sudo
   gather_facts: true
   tasks:
+
```
Comment (eleo007) [inline molecule/pxc/playbooks/prepare.yml:8]: Centos6 part can be removed as not supported.

File: molecule/pxc/pxc80-bootstrap/molecule/centos-6/molecule.yml [hunk 4]
```diff
@@ -10,7 +10,7 @@ lint:
 platforms:
```
Comment (eleo007) [inline molecule/pxc/pxc80-bootstrap/molecule/centos-6/molecule.yml:10]: remove centos6 as not supported.

File: molecule/pxc/pxc80-bootstrap/molecule/debian-9/molecule.yml [hunk 5]
```diff
@@ -10,11 +10,11 @@ lint:
 platforms:
```
Comment (eleo007) [inline molecule/pxc/pxc80-bootstrap/molecule/debian-9/molecule.yml:10]: remove debian-9

File: molecule/pxc/pxc80-bootstrap/molecule/ubuntu-xenial/molecule.yml [hunk 6]
```diff
@@ -10,10 +10,10 @@ lint:
 platforms:
```
Comment (eleo007) [inline molecule/pxc/pxc80-bootstrap/molecule/ubuntu-xenial/molecule.yml:10]: Remove ubuntu-xenial

File: molecule/pxc/pxc80-common/molecule/centos-6/molecule.yml [hunk 7]
```diff
@@ -10,18 +10,18 @@ lint:
 platforms:
```
Comment (eleo007) [inline molecule/pxc/pxc80-common/molecule/centos-6/molecule.yml:10]: Clean up unsupported versions here too.

**Review by panchal-yash — COMMENTED** [review]
molecule/pxc/run-local.sh was for local testing using vagrant and is not included in any of the playbooks.

**Pull request merged**